src/configuration/FirestoreConfigurationContext.tsx

```diff
@@ -37,6 +39,7 @@ const defaultConfigurationContextState: ConfigurationContextState = {
   userProfiles: [],
   modesWeSellTicketsFor: [],
   paymentTypes: [],
+  vatPercent: 0,
```

Comment:
The defaults (like `defaultVatPercent`) should/could probably also be used here? If so then also update the other variables.